### PR TITLE
Schedule downstream footprint tasks

### LINKED ISSUE #250: Incorporate downstream tasks in building footprints pipeline
With #233 more-or-less complete, we should be able add a couple of downstream tasks to the pipeline, making it more "production" grade.

- After the loading step is done, we can trigger the dbt build of the footprints-with-tiger-data. We could either use dbt core for this, or a dbt cloud operator.
- After the dbt build is done, we can trigger the unload-to-s3 step.

### PR DESCRIPTION
**Pull request by ian-r-rose**
Fixes #250 

Previously, we were only scheduling the data load of building footprints. The dbt build was left for a "nightly" job, and the data unload to geoparquet/shapefile was done manually.

This incorporates the dbt build for footprints as well as the data unloading step into the Airflow dag so that it's more of an end-to-end pipeline. I'm triggering the dbt build using the `DbtCloudJobRunOperator`, which talks directly to environments and jobs in dbt cloud. We could also do this with dbt core, and reworking things to do that would be an interesting exercise. For the moment, however, using dbt cloud was simpler.

### CODE REVIEW

File: .github/workflows/submit-job.yml
Comment (ian-r-rose) [inline .github/workflows/submit-job.yml:1]: This is a drive-by fix of this github action, which was inadvertently broken by #257. The action here is more of a demonstration/smoke-test of submitting batch jobs via github actions, and could be deleted.

File: airflow/dags/geo_reference/load_building_footprints.py [hunk 1]
```diff
@@ -9,6 +9,24 @@
 from airflow.decorators import dag
 from airflow.providers.amazon.aws.operators.batch import BatchOperator
 from airflow.providers.amazon.aws.sensors.batch import BatchSensor
+from airflow.providers.dbt.cloud.operators.dbt import DbtCloudRunJobOperator
+
+
+def _construct_batch_args(name: str, command: list[str]) -> dict:
```
Comment (ian-r-rose) [inline airflow/dags/geo_reference/load_building_footprints.py:15]: Small refactor to try to avoid repeating the same batch job args over and over

File: terraform/aws/modules/infra/batch.tf [hunk 2]
```diff
@@ -84,6 +84,11 @@ resource "aws_iam_role_policy_attachment" "s3_scratch_policy_role_attachment" {
   policy_arn = aws_iam_policy.s3_scratch_policy.arn
 }
 
+resource "aws_iam_role_policy_attachment" "dof_demographics_read_write_access" {
```
Comment (ian-r-rose) [inline terraform/aws/modules/infra/batch.tf:87]: This gives our Batch ECS tasks write access to the DOF demographics bucket

**Review by melanie-logan — APPROVED** [review]
Updates look good Ian!

**Comment by ian-r-rose** [comment]
Thanks for the review @melanie-logan!
